aioasuswrt/connection.py

```diff
@@ -118,7 +117,13 @@ async def async_run_command(self, command, first_try=True):
                 return []
         except TimeoutError:
             _LOGGER.error("Host timeout.")
-            return []
+            async with self._io_lock:
+                self.disconnect()
+            if first_try:
+                _LOGGER.info("Trying one more time")
```

Comment:
My preference would be to make this a debug statement, an info statement might be added to the `else` actually, as that is even more interesting to the user.

Reply:
Sure, done